doc/migration/version-upgrade-guide.md

```diff
@@ -335,6 +335,138 @@ Upon updating the project to `v0.8.2` the following breaking changes apply:
   replace github.com/operator-framework/operator-sdk => github.com/operator-framework/operator-sdk v0.9.0
   ```
 
+## `v0.10.x`
+
+- The scorecard configuration format for the `operator-sdk scorecard` command has changed. See [`doc/test-framework/scorecard`](../test-framework/scorecard.md) for more info.
+- The CSV config field `role-path` is now `role-paths` and takes a list of strings. To migrate:
+    ```yaml
+    # Old
+    role-path: path/to/role.yaml
+    # New
+    role-paths:
+    - path/to/role.yaml
+    ```
```

Comment:
```suggestion
* Replace:
    ```yaml
    role-path: path/to/role.yaml
    ```
* For
    ```yaml
    role-paths:
    - path/to/role.yaml
    ```
```